content/applications/services/helpdesk/overview/receiving_tickets.rst

```diff
@@ -206,72 +147,129 @@ of when a :guilabel:`URL Regex` action is triggered (e.g., a page visit).
 Edit existing rules, or create a new one by clicking :guilabel:`Add a line`, and fill out the pop-up
 form details based on how the rule should apply.
 
-If a :guilabel:`Chatbot` will be included on this channel, select it from the dropdown. If the
-chatbot will only be active when no operators are available, check the box labeled
-:guilabel:`Enabled only if no operator`.
+To include a :guilabel:`Chatbot` on this channel, select it from the drop-down menu. If the chatbot
+should only be active when no operators are available, check the box labeled :guilabel:`Enabled only
+if no operator`.
 
 .. note::
-   If a chatbot is added to a live chat channel, then 3 new smart buttons will appear on the channel
-   settings form: :guilabel:`Chatbots`, :guilabel:`Sessions`, and :guilabel:`% Happy`.
-
-   - The :guilabel:`Chatbots` smart button is where the chatbot can be programmed with a
-     :guilabel:`Script`. Each line in the :guilabel:`Script` contains a :guilabel:`Message`,
-     :guilabel:`Step Type`, :guilabel:`Answers`, and conditional :guilabel:`Only If` logic that
-     applies when certain pre-filled answers are chosen. To create more steps in the
-     :guilabel:`Script`, click :guilabel:`Add a line` and fill out the script steps form according
-     to the desired logic.
-   - :guilabel:`Sessions` is where live chat sessions are recorded in order of descending
-     :guilabel:`Session Date`, by default. Each record includes the :guilabel:`Attendees` involved
-     in the live chat session, the :guilabel:`# Messages`, as well as any :guilabel:`Rating` that
-     was received when the session ended.
-   - The :guilabel:`% Happy` smart button includes a log of ratings that were left by live chat
-     attendees, and are labeled by date, time, and the support agent who was responsible for the
-     live chat session.
-
-Add the URL for the pages this channel will be applied to in the :guilabel:`URL Regex` field. If
-this channel will only be available to users in specific countries, add them to the
+   If a chatbot is added to a live chat channel, a new :guilabel:`Chatbots` smart button appears on
+   the channel settings form. Click here to create and update the chatbot *script*. Each line in the
+   script contains a :guilabel:`Message`, :guilabel:`Step Type`, :guilabel:`Answers`, and
+   conditional *Only If* logic that applies when certain pre-filled answers are chosen. To create
+   more steps in the script, click :guilabel:`Add a line` and fill out the script steps form
+   according to the desired logic.
```

Comment:
Some clarification is needed here (maybe add a screen shot of the smart button). 

People assume chatbots are just full of AI